Assignment 2.Rmd

```diff
@@ -87,12 +94,53 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+```{r}
+EDGE.st <- EDGE %>% group_by(to) %>% summarize(sum(count))
+names(EDGE.st) <- c("id", "count")
+EDGE.tc <- EDGE %>% group_by(from) %>% summarise(sum(count))
+EDGE.tc
+names(EDGE.tc) <- c("id", "count")
+total <- bind_rows(EDGE.st, EDGE.tc)
+VERTEX2 <- full_join(VERTEX, total, by = "id")
+g2 <- graph.data.frame(EDGE, directed = TRUE, vertices = VERTEX2)
+plot(g2, layout = layout.fruchterman.reingold, vertex.color = VERTEX2$gender)
+```
```

Comment:
It looks like you have created the values to size the vertices with successfully, but didn't employ them in your graph? The vertices should be different sizes (`vertex.size = VERTEX2$sum(count)`